packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -79,11 +84,36 @@ const MyAdsRowRenderer = observer(({ row: advert }) => {
     const handleOnEdit = () =>
         enable_action_point && floating_rate_store.rate_type !== rate_type ? onClickSwitchAd() : onClickEdit();
 
+    const should_show_tooltip_icon =
+        (visibility_status?.length === 1 && visibility_status?.[0] !== 'advert_inactive') ||
+        visibility_status?.length > 1;
+
+    const getErrorCodes = () => {
+        let updated_visibility_status = [...visibility_status];
+        if (!is_ads_listed && !updated_visibility_status.includes('advertiser_ads_paused'))
+            updated_visibility_status = [...updated_visibility_status, 'advertiser_ads_paused'];
+        if (!enable_action_point && updated_visibility_status.includes('advert_inactive'))
+            updated_visibility_status = updated_visibility_status.filter(status => status !== 'advert_inactive');
+        return updated_visibility_status;
```

Comment:
why do we need to update `visibility_status`?

Reply:
to handle the toggle ad pause, since the api response is not updated, visibility status has to be manually updated to include the advertiser_ads_paused field.